### PR TITLE
🌱 add validation for CRD

### PR DESCRIPTION
**Pull request by jasonliu747**
Signed-off-by: Jason Liu <jasonliu747@gmail.com>

### Ⅰ. Describe what this PR does
add some range validation for fields.

### CODE REVIEW

**Review by shinytang6 — APPROVED** [review]
🌞 lgtm

File: apis/slo/v1alpha1/nodeslo_types.go [hunk 1]
```diff
@@ -189,19 +195,21 @@ type ResctrlQoS struct {
...
 	CFSQuotaBurstOnly CPUBurstPolicy = "cfsQuotaBurstOnly"
-	// enable both
+	// CPUBurstAuto enable both
 	CPUBurstAuto CPUBurstPolicy = "auto"
 )
 
 type CPUBurstConfig struct {
 	Policy CPUBurstPolicy `json:"policy,omitempty"`
 	// cpu burst percentage for setting cpu.cfs_burst_us, legal range: [0, 10000], default as 1000 (1000%)
+	// +kubebuilder:validation:Maximum=1000
```
Comment (hormes) [inline apis/slo/v1alpha1/nodeslo_types.go:211]: maximum is 10000？

Comment (jasonliu747) [reply apis/slo/v1alpha1/nodeslo_types.go]: nice catch, fixed.

File: apis/slo/v1alpha1/nodeslo_types.go [hunk 2]
```diff
@@ -189,19 +195,21 @@ type ResctrlQoS struct {
 type CPUBurstPolicy string
 
 const (
-	// disable cpu burst policy
+	// CPUBurstNone disable cpu burst policy
```
Comment (saintube) [inline apis/slo/v1alpha1/nodeslo_types.go:198]: `disable` -> `disables`. the same below

Comment (jasonliu747) [reply apis/slo/v1alpha1/nodeslo_types.go]: @saintube done, Mr. Leeuwenhoek

**Pull request merged**